pkg/kubectl/cmd/util/factory_client_access.go

```diff
@@ -477,8 +477,8 @@ const (
 	SecretForDockerRegistryV1GeneratorName = "secret-for-docker-registry/v1"
 	SecretForTLSV1GeneratorName            = "secret-for-tls/v1"
 	ConfigMapV1GeneratorName               = "configmap/v1"
-	ClusterRoleBindingV1GeneratorName      = "clusterrolebinding.rbac.authorization.k8s.io/v1alpha1"
-	RoleBindingV1GeneratorName             = "rolebinding.rbac.authorization.k8s.io/v1alpha1"
```

Comment:
if that's the path we take, then we need to:
* rename these constants to v1alpha1
* create a second set of constants for v1beta1
* duplicate the generators and tweak them to create v1beta1 objects
* add discovery switching to select the default generator